pysr/_cli/main.py

```diff
@@ -0,0 +1,16 @@
+import click
+from ..julia_helpers import install
+
+
+@click.group("cli")
+@click.pass_context
+def cli(context):
+    ctx = context
+
+
+@cli.command("install", help="Install required Julia dependencies")
+@click.option("--julia_project", default=None)
+@click.option("--quiet", default=False)
+@click.option("--precompile", default=None)
```

Comment:
I think this should be a feature flag: https://click.palletsprojects.com/en/8.1.x/options/#feature-switches

where `--precompile` => `True`, `--no-precompile` => `False`, and otherwise => `None`.

Reply:
Thank you I didn't realise there were three different inputs for precompile.